src/i18n/DE/grid-resources.ts

```diff
@@ -0,0 +1,132 @@
+import { IGridResourceStrings } from 'igniteui-angular';
+
+// exported below as re-cast to create declaration type with expanded properties
+const GridResourceStringsDE_: ExpandRequire<IGridResourceStrings> = {
+  igx_grid_groupByArea_message: 'Lege eine Spaltenüberschrift hier ab, um danach zu sortieren.',
+  igx_grid_emptyFilteredGrid_message: 'Keine Einträge gefunden.',
+  igx_grid_emptyGrid_message: 'Grid hat keine Daten.',
+  igx_grid_filter: 'Filter',
+  igx_grid_filter_row_close: 'Schließen',
+  igx_grid_filter_row_reset: 'Zurücksetzen',
+  igx_grid_filter_row_placeholder: 'Füge Filterwert hinzu',
```

Comment:
Phrasing seems odd but it depends on how it's used. If that's a menu item/command, I'd expect something like "Filterwert hinzufügen..."

Reply:
The original text is "Add filter value". It is shown in an input field as a placeholder when the user has not entered a value yet.